addon/locales/en-us/config.js

```diff
@@ -0,0 +1,16 @@
+// Ember-I18n includes configuration for common locales. Most users
+// can safely delete this file. Use it if you need to override behavior
+// for a locale or define behavior for a locale that Ember-I18n
+// doesn't know about.
+export default {
```

Comment:
Technically I think `ember-i18n` requires the file to be present, even if we don't set any options.  (I got errors when I omitted this file)

I debated about leaving the commented out text (from the template) in place, but decided it might be useful sometime; documentation is a smidge thin.